### PR TITLE
add dolly-v2-12b for oss

### PR DESCRIPTION
**Pull request by novaturient95**
### Validation results

**run to save model to blobstoage:** [run](https://ml.azure.com/runs/gentle_garden_9yjcf8d4ph?wsid=/subscriptions/80c77c76-74ba-4c8c-8229-4c3b2957990c/resourcegroups/huggingface-registry-test1/workspaces/test-eastus&tid=72f988bf-86f1-41af-91ab-2d7cd011db47#/?graphId=4eb95658-dfb7-422e-b028-a58229cfd624&label=save_model_to_blobstorage+databricks-dolly-v2-12b&path=%2Fruns%2Fgentle_garden_9yjcf8d4ph&runId=gentle_garden_9yjcf8d4ph)

**batch validation results:** [run](https://ml.azure.com/runs/batchjob-b3d6e706-a9b1-fe48-da0c-708425786ac7?wsid=/subscriptions/80c77c76-74ba-4c8c-8229-4c3b2957990c/resourceGroups/huggingface-registry-test1/providers/Microsoft.MachineLearningServices/workspaces/test-eastus&tid=72f988bf-86f1-41af-91ab-2d7cd011db47#)

**online validation results**

ep: [link](https://ml.azure.com/endpoints/realtime/dolly-oss-deploy-di/test?wsid=/subscriptions/80c77c76-74ba-4c8c-8229-4c3b2957990c/resourceGroups/huggingface-registry-test1/providers/Microsoft.MachineLearningServices/workspaces/test-eastus&tid=72f988bf-86f1-41af-91ab-2d7cd011db47)

image

**MLModel:** 

```
flavors:
  python_function:
    env:
      conda: conda.yaml
      virtualenv: python_env.yaml
    loader_module: mlflow.transformers
    model_binary: model
    python_version: 3.8.18
  transformers:
    code: null
    components:
    - tokenizer
    framework: pt
    instance_type: InstructionTextGenerationPipeline
    model_binary: model
    pipeline_model_type: GPTNeoXForCausalLM
    task: text-generation
    tokenizer_type: GPTNeoXTokenizerFast
    transformers_version: 4.35.2
metadata:
  base_model_name: databricks/dolly-v2-12b
  base_model_task: text-generation
  is_acft_model: true
  is_finetuned_model: false
mlflow_version: 2.8.1
model_size_bytes: 47369790595
model_uuid: f7383acbad6f48069c76424fe00f52fd
signature:
  inputs: '[{"type": "string"}]'
  outputs: '[{"type": "string"}]'
  params: '[{"name": "top_p", "type": "float", "default": 0.9, "shape": null}, {"name": "temperature", "type": "float", "default": 0.2, "shape": null}, {"name": "max_new_tokens", "type": "integer", "default": 50, "shape": null}, {"name": "do_sample", "type": "boolean", "default": true, "shape": null}, {"name": "return_full_text", "type": "boolean", "default": true, "shape": null}]'
utc_time_created: '2023-12-07 04:44:00.270496'
```

**conda.yaml**

```
channels:
- anaconda
- conda-forge
dependencies:
- python=3.8.18
- pip:
  - accelerate==0.25.0
  - cffi==1.16.0
  - dill==0.3.7
  - google-api-core==2.14.0
  - mlflow==2.8.1
  - numpy==1.23.5
  - packaging==23.2
  - pillow==10.0.1
  - protobuf==3.20.3
  - pyyaml==6.0.1
  - requests==2.31.0
  - scikit-learn==1.1.3
  - scipy==1.10.1
  - sentencepiece==0.1.99
  - torch==1.13.1
  - torchvision==0.14.1
  - transformers==4.34.1
- pycocotools=2.0.4
name: project_environment
```

### CODE REVIEW

File: assets/models/system/databricks-dolly-v2-12b/spec.yaml [hunk 1]
```diff
@@ -4,48 +4,19 @@ path: ./
...
-  - Standard_ND96amsr_A100_v4
-  finetune_computes_allow_list:
-  - Standard_ND40rs_v2
-  - Standard_ND96asr_v4
-  - Standard_NC24ads_A100_v4
-  - Standard_NC48ads_A100_v4
-  - Standard_NC96ads_A100_v4
-  - Standard_ND96amsr_A100_v4
+  author: Databricks, Inc.
+  license: apache-2.0
```
Comment (novaturient95) [inline assets/models/system/databricks-dolly-v2-12b/spec.yaml:14]: HF has MIT but GH has apache-2.0

https://github.com/databrickslabs/dolly/blob/master/LICENSE

File: assets/models/system/databricks-dolly-v2-12b/asset.yaml [hunk 2]
```diff
@@ -1,4 +1,4 @@
 extra_config: model.yaml
 spec: spec.yaml
 type: model
-categories: ["Foundation Models"]
\ No newline at end of file
+categories: ["Foundation Models", "OSS"]
```
Comment (novaturient95) [inline assets/models/system/databricks-dolly-v2-12b/asset.yaml:4]:
```suggestion
categories: ["Foundation Models", "MLflow Transformers"]
```

File: assets/models/system/databricks-dolly-v2-12b/description.md [hunk 3]
```diff
@@ -40,50 +42,29 @@ databricks/dolly-v2-7b|	0.392|	0.633838|	0.607735|	0.686517|	0.406997|	0.750816|
...
-Text generation | Text generation | <a href="https://huggingface.co/datasets/cnn_dailymail" target="_blank"> cnn_dailymail </a> | <a href="https://aka.ms/azureml-eval-sdk-text-generation/" target="_blank">evaluate-model-text-generation.ipynb</a> | <a href="https://aka.ms/azureml-eval-cli-text-generation/" target="_blank">evaluate-model-text-generation.yml</a>
-
 # Inference samples
 
-Inference type|Python sample (Notebook)|CLI with YAML
-|--|--|--|
-Real time|<a href="https://aka.ms/azureml-infer-online-sdk-text-generation-dolly" target="_blank">text-generation-online-endpoint-dolly.ipynb</a>|<a href="https://aka.ms/azureml-infer-online-cli-text-generation-dolly" target="_blank">text-generation-online-endpoint-dolly.sh</a>
-Batch |<a href="https://aka.ms/azureml-infer-batch-sdk-text-generation" target="_blank">text-generation-batch-endpoint.ipynb</a>| coming soon
-
 ## Sample inputs and outputs (for real-time inference)
```
Comment (novaturient95) [inline assets/models/system/databricks-dolly-v2-12b/description.md:47]: H1

Comment (novaturient95) [inline assets/models/system/databricks-dolly-v2-12b/description.md:47]:
```suggestion
# Sample inputs and outputs (for real-time inference)
```

Comment (novaturient95) [inline assets/models/system/databricks-dolly-v2-12b/description.md:47]:
```suggestion
# Sample inputs and outputs
```

File: assets/models/system/databricks-dolly-v2-12b/spec.yaml [hunk 4]
```diff
@@ -4,48 +4,19 @@ path: ./
...
-  - Standard_NC96ads_A100_v4
-  - Standard_ND96amsr_A100_v4
-  finetune_computes_allow_list:
-  - Standard_ND40rs_v2
-  - Standard_ND96asr_v4
-  - Standard_NC24ads_A100_v4
-  - Standard_NC48ads_A100_v4
-  - Standard_NC96ads_A100_v4
-  - Standard_ND96amsr_A100_v4
+  author: Databricks, Inc.
```
Comment (novaturient95) [inline assets/models/system/databricks-dolly-v2-12b/spec.yaml:13]:
```suggestion
  author: databricks
```

File: assets/models/system/databricks-dolly-v2-12b/spec.yaml [hunk 5]
```diff
@@ -3,49 +3,19 @@ name: databricks-dolly-v2-12b
...
-  - Standard_NC96ads_A100_v4
-  - Standard_ND96amsr_A100_v4
-  finetune_computes_allow_list:
-  - Standard_ND40rs_v2
-  - Standard_ND96asr_v4
-  - Standard_NC24ads_A100_v4
-  - Standard_NC48ads_A100_v4
-  - Standard_NC96ads_A100_v4
-  - Standard_ND96amsr_A100_v4
+  author: Databricks
```
Comment (novaturient95) [inline assets/models/system/databricks-dolly-v2-12b/spec.yaml:12]: Keeping it here?

**Review by tinaem — APPROVED** [review]

File: assets/models/system/databricks-dolly-v2-12b/description.md [hunk 6]
```diff
@@ -40,50 +20,42 @@ databricks/dolly-v2-7b|	0.392|	0.633838|	0.607735|	0.686517|	0.406997|	0.750816|
...
 }
 ```
 
 ### Sample output
 ```json
 [
-    {
-        "0": "I really enjoy wasabi mayo. The spice found in wasabi is a perfect compliment to the mayo's sweetness."
-    }
+  "I believe the meaning of life is to find what you love and do it very well. I love to write, I love to paint, I love to sing, I love to dance, I love to act, I love to direct, I love to play all kinds of music, I love to see beautiful things, I love to give beautiful things, and I love to have beautiful experiences.\n life is a journey not a destination. so enjoy the ride.\n\nI am a very positive person. I believe that you should"
```
[the remark this answers is not in the document]
Comment (novaturient95) [reply assets/models/system/databricks-dolly-v2-12b/description.md]: updated

**Review by BhavanaTumma — APPROVED** [review]
Approved

**Pull request merged**